RadixWallet/Core/FeaturePrelude/AddressView/AddressDetails+View.swift

```diff
@@ -0,0 +1,238 @@
+extension AddressDetails.State {
+	var viewState: AddressDetails.ViewState {
+		.init(
+			address: address,
+			title: title,
+			qrImage: qrImage,
+			showEnlarged: showEnlarged,
+			showShare: showShare
+		)
+	}
+}
+
+// MARK: - AddressDetails.View
+
+public extension AddressDetails {
+	struct ViewState: Equatable {
+		let address: LedgerIdentifiable.Address
+		let title: Loadable<String?>
+		let qrImage: Loadable<CGImage>
+		let showEnlarged: Bool
+		let showShare: Bool
+	}
+
+	@MainActor
+	struct View: SwiftUI.View {
+		private let store: StoreOf<AddressDetails>
+
+		public init(store: StoreOf<AddressDetails>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+				content(viewStore)
+					.withNavigationBar {
+						viewStore.send(.closeButtonTapped)
+					}
+					.presentationDetents([.large])
+					.presentationDragIndicator(.visible)
+					.task {
+						viewStore.send(.task)
+					}
+					.sheet(isPresented: viewStore.binding(get: \.showShare, send: .shareDismissed)) {
+						ShareView(items: [viewStore.address.address])
+					}
+			}
+		}
+
+		private func content(_ viewStore: ViewStoreOf<AddressDetails>) -> some SwiftUI.View {
+			VStack(spacing: .zero) {
+				top(title: viewStore.title, showQrCode: viewStore.showQrCode, qrImage: viewStore.qrImage)
+
+				fullAddress(address: viewStore.colorisedAddress)
+					.padding(.vertical, .medium3)
+
+				bottom(showVerifyOnLedger: viewStore.showVerifyOnLedger)
+
+				Spacer()
+			}
+			.multilineTextAlignment(.center)
+			.padding([.horizontal, .bottom], .medium3)
+			.overlay(alignment: .top) {
+				Group {
+					if viewStore.showEnlarged {
+						enlargedView(text: viewStore.enlargedText)
+					}
+				}
+				.animation(.easeInOut, value: viewStore.showEnlarged)
+			}
+		}
+
+		private func top(title: Loadable<String?>, showQrCode: Bool, qrImage: Loadable<CGImage>) -> some SwiftUI.View {
+			VStack(spacing: .small2) {
+				loadable(title) {
+					ProgressView()
+				} successContent: { title in
+					Text(title)
+						.textStyle(.sheetTitle)
+						.foregroundColor(.app.gray1)
+				}
+
+				if showQrCode {
+					Text("Address QR Code")
```

Comment:
Will update & use localisaton once https://github.com/radixdlt/apps-localization-src/pull/367 gets merged

Reply:
We usually put `// FIXME: Strings` on these so that we can find them later in case we forget